std/math.d

```diff
@@ -4812,62 +4919,88 @@ long lround(real x) @trusted nothrow @nogc
         assert(0, "lround not implemented");
 }
 
-version(Posix)
+///
+@safe nothrow @nogc unittest
 {
-    @safe nothrow @nogc unittest
+    version(Posix)
     {
         assert(lround(0.49) == 0);
         assert(lround(0.5) == 1);
         assert(lround(1.5) == 2);
     }
 }
 
-/****************************************************
- * Returns the integer portion of x, dropping the fractional portion.
- *
- * This is also known as "chop" rounding.
- */
-real trunc(real x) @trusted nothrow @nogc
+version(StdDdoc)
```

Comment:
Ok, so apparently `ret` in ASM doesn't work with an `auto` return type. So we now have this mess.